src/sagas/ui.js

```diff
@@ -25,6 +28,20 @@ export function* projectSuccessfullySaved() {
   yield put(hideSaveIndicator());
 }
 
+export function* assignmentCreated({payload: {assignment}}) {
+  if (assignment.alternateLink) {
+    yield put(assignmentDisplayed({
+      url: assignment.alternateLink,
+      exportType: 'assignment',
+    }));
+  } else {
+    yield put(assignmentDisplayed({
+      url: 'https://classroom.google.com/',
```

Comment:
Would it be possible to at least direct-link to the course in this case?

Reply:
This is what the payload returned looks like for a draft assignment.
```
assigneeMode: "ALL_STUDENTS"
associatedWithDeveloper:true
courseId:"2022089786"
creationTime:"2018-09-09T16:51:45.383Z"
creatorUserId:"xxxxxxxxxxxxxxxxxxxxxxxx"
dueDate:{year: 2018, month: 9, day: 11}
dueTime:{hours: 16}
id:"17715559980"
materials:[{…}]
state:"DRAFT"
submissionModificationMode:"MODIFIABLE_UNTIL_TURNED_IN"
title:"Page Title"
updateTime:"2018-09-09T16:51:45.325Z"
workType:"ASSIGNMENT"
```
The payload has a courseID as a string of digits, but the url to direct to the classroom looks like this `https://classroom.google.com/u/0/c/MjAyMjA4OTc4Nlpa`
where the id is a string of letters of digit. I'm not quite sure if you can extract this URL with the information given.
My only other thought is that we might be able store the classroom url when retrieving the lists of courses and pass that url along when submiting the assignment. A bit annoying but may be worth it?

Reply:
Mm, good call on just storing the course URL (weird that the ID and public-facing URL seem unrelated). Seems pretty straightforward since we are already storing course info? But if it’s annoying totally reasonable as a followup as well.